reporters/GlobalSummaryReporter.js

```diff
@@ -31,7 +31,7 @@ class GlobalSummaryReporter {
 
     this.web3 = this.empEventClient.web3;
     this.toBN = this.web3.utils.toBN;
-    this.toWei = this.web3.utils.toBN;
+    this.toWei = this.web3.utils.toWei;
```

Comment:
Was this really the issue with the GCR? lol, I'm amazed this didn't break tons of things.

Reply:
I cant believe this got through and didn't break all the things 😆 it looks like the only places we were using it were for scaling.